tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1369,6 +1369,59 @@ def test_get_privacy_requests_sets_days_left(
         data = response.json()["items"][0]
         assert data["days_left"] == days_left
 
+    @mock.patch(
+        "fidesops.ops.service.privacy_request.request_runner_service.run_privacy_request.delay"
+    )
+    def test_sort_privacy_request_by_due_date(
+        self,
+        run_access_request_mock,
+        generate_auth_header,
+        url,
+        db,
+        api_client: TestClient,
+        policy: Policy,
+    ):
+        days_left_values = []
+        data = []
+        now = datetime.utcnow()
+        for i in range(0, 10):
+            requested_at = now + timedelta(days=i)
+            data.append(
+                {
+                    "requested_at": str(requested_at),
+                    "policy_key": policy.key,
+                    "identity": {"email": "test@example.com"},
+                }
+            )
+            days_left_values.append(i + policy.execution_timeframe)
+
+        api_client.post(url, json=data)
+
+        auth_header = generate_auth_header(scopes=[PRIVACY_REQUEST_READ])
+        resp = api_client.get(
+            f"{url}?sort_direction=asc&sort_field=due_date",
```

Comment:
Does this conform to the spec of the Enum?

Reply:
Yes, it does. The value that is passed in for `sort_direction` is validated by fastapi, or possibly pydantic, to ensure that it is one of the enumerations.

Reply:
Here is what's return if an invalid option is used
```json
{
    "detail": [
        {
            "loc": [
                "query",
                "sort_direction"
            ],
            "msg": "value is not a valid enumeration member; permitted: 'desc', 'asc'",
            "type": "type_error.enum",
            "ctx": {
                "enum_values": [
                    "desc",
                    "asc"
                ]
            }
        }
    ]
}
```